github-orgs/kubeflow/org.yaml

```diff
@@ -1005,11 +1005,12 @@ orgs:
               pytorch-operator: admin
           third-party-bots:
             description: Team for third party bots
-            members:
+            maintainers:
```

Comment:
Why is this changing at the same time?

Reply:
Because there were a weird sync issue we notice before https://github.com/kubeflow/internal-acls/pull/362, we're still not sure if this is the root-cause, but it won't hurt